assets/tests/batch_check_tests.yaml

```diff
@@ -0,0 +1,94 @@
+tests:
```

Comment:
I know its captured in another test but would be good to have an explicit case where you have the same check with two unique correlation IDs.

Reply:
That scenario is currently not being accounted for by batch check. If a user sends us 10 identical checks, each with a different correlation_id, we will process them all. [There is a ticket for it as follow up](https://auth0team.atlassian.net/browse/DXAZC-2712)

Reply:
Correct. And that's what I expect with this first iteration. But it may not be intuitive what the desired behavior of that should be. Encoding that in a test may be helpful 🤷‍♂️ .